integration_tests/steps/integration_tests.py

```diff
@@ -569,6 +569,31 @@ def test_output(context, scenario, sim_duration, slot_length, tick_length):
         assert all([permanent_load.strategy.energy == ei for ei in energy_profile])
 
 
+@then('we test energy bills')
+def test_energy_bills(context):
```

Comment:
I know it is not on the scope of this story, but it would be great to add the corresponding test for the prices, to validate whether the "spent" and "earned" values are populated correctly.